doc/source/analysis.rst

```diff
@@ -148,6 +148,17 @@ Let's tell runsample to put our analysis into a directory called 947 and also te
 
 You can see from the output that the sample started and finished. If there were errors, they would show up in between those two lines and you would have to view the :doc:`help` documentation.
 
+Run with fasta files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Simply run runsample with the --fasta argument: 
+
+.. code-block:: bash
+
+     $> runsample -od 947 --fasta path/to/fasta/files ngs_mapper/tests/fixtures/functional/947.ref.fasta 947 
```

Comment:
Seems like a lot of work just to ensure documentation is easy to follow, but it is probably worth it in the long run.